src/reporters/json.ts

```diff
@@ -483,13 +483,17 @@ export default class JSONReporter extends BaseReporter {
           });
         }
         if (browserconsole) {
-          browserconsole.forEach(({ timestamp, text, type, step }) => {
+          browserconsole.forEach(({ timestamp, text, type, step, error }) => {
             this.writeJSON({
               type: 'journey/browserconsole',
               journey,
               timestamp,
               step,
-              payload: { text, type } as Payload,
+              payload: {
```

Comment:
you can pass error as a standalone field here. will be formatted correctly in write handler.

Reply:
write handler will get only text from the error, no? if we want the stack trace implicitely we will have to get it like it's done here...

Reply:
The payload would contain `type and text` as it was previously and you would write `errror` in top level which would be picked up here - https://github.com/elastic/synthetics/blob/8c502cfa027781f4031829d97763ac3ad3496cf4/src/reporters/json.ts#L589

Reply:
got it, i have pushed the update